src/imager.jsx

```diff
@@ -79,9 +82,39 @@ module.exports = function (config) {
       }
     },
 
+    getLowSrc: function () {
+      var lowSrc = this.props.src;
+
+      // use the defined low src property
+      if (this.props.lowSrc) {
+        lowSrc = this.props.lowSrc;
+      }
+      /*// use imagers gif src
+      else if (imgr) {
+        lowSrc = imgr.gif.src;
+      }*/
+      // use the smallest width and replace it in the default src property
+      else {
+        var smallestWidth = -1;
+
+        if (config.availableWidths && config.availableWidths.length) {
+          for (var i = 0, l = config.availableWidths.length; i < l; i++) {
+            var width = config.availableWidths[i];
+            if (smallestWidth === -1 || smallestWidth > width) {
+              smallestWidth = width;
+            }
+          }
+        }
+
+        lowSrc = this.props.src.replace('{width}', smallestWidth);
+      }
+
+      return lowSrc;
+    },
+
     getImageSrc: function () {
       if (!this.state.width) {
-        return imgr.gif.src;
+        return this.getLowSrc();
```

Comment:
The width is currently [calculated](https://github.com/joernroeder/Imager.jsx/blob/master/src/imager.jsx#L126) via the imager module at and updated [afterwards](https://github.com/joernroeder/Imager.jsx/blob/master/src/imager.jsx#L129). Using `lowScr` (the smallest version) whenever we don't have any information about the width tries to follow the mobile first appoach.
Alternatively we could shim the `imgr.gif.scr` property and use a transparent gif, but we have to render the same DOM on the client and the server to get the full advantage of server-side rendering and serving the page without images from the server isn't a good idea…